taca/backup/backup.py

```diff
@@ -119,6 +119,8 @@ def _get_run_type(self, run):
         try:
             if "ST-" in run:
                 run_type = "hiseqx"
+            elif "_A":
```

Comment:
Hi Senthil, do you mean `"_A" in run`? Also I'm thinking about that `_A` matched with the general pattern of flowcell position A, such as "180919_D00415_0368_AHKNM3BCX2"  which is a hiseq run.

Reply:
Good catch @chuan-wang 😉  It should be `elif "_A0" in run`, I will fix it. This is why one should not make a PR when in a hurry to leave 😅